common.blocks/link/link.en.md

```diff
@@ -161,6 +162,79 @@ Sets the size value for all types of links.
 }
 ```
 
+<a name="view"></a>
+
+#### `view` modifier
+
+Acceptable values: `'minor'`, `'external'`, `'ghost'`, `'text'`, `'strong'`.
+
+Use case: `BEMJSON`.
+
+##### Secondary link (`view` modifier with `minor` value)
+
+The modifier visually highlights secondary links on a page.
+
+```js
+{
+    block : 'link',
+    mods : { theme : 'islands', size: 'm', view : 'minor' },
+    url : 'https://bem.info/',
+    content : 'bem.info'
+}
+```
+
+##### External link (`view` modifier with `external` value)
+
+The modifier visually highlights external links on a page.
+
+```js
+{
+    block : 'link',
+    mods : { theme : 'islands', size: 'm', view : 'external' },
+    url : 'https://bem.info/',
+    content : 'bem.info'
+}
+```
+
+##### Ghost link (`view` modifier with `ghost` value)
+
+The modifier changes visual representation of the link. For example, use it if you do not need to focus attention on the link:
```

Comment:
The modifier changes the way the link looks. Use it if you don't want the link to stand out on the page.